src/GovUk.Education.ExploreEducationStatistics.Content.Services/GlossaryService.cs

```diff
@@ -0,0 +1,52 @@
+#nullable enable
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Common.Cache;
+using GovUk.Education.ExploreEducationStatistics.Content.Model.Database;
+using GovUk.Education.ExploreEducationStatistics.Content.Services.Cache;
+using GovUk.Education.ExploreEducationStatistics.Content.Services.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Content.Services.ViewModels;
+using Microsoft.EntityFrameworkCore;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Services
+{
+    public class GlossaryService : IGlossaryService
+    {
+        private readonly ContentDbContext _contentDbContext;
+
+        public GlossaryService(ContentDbContext contentDbContext)
+        {
+            _contentDbContext = contentDbContext;
+        }
+
+        [BlobCache(typeof(GlossaryCacheKey))]
+        public async Task<List<GlossaryCategoryViewModel>> GetAllGlossaryEntries()
+        {
+            var categories = "ABCDEFGHIJKLMNOPQRSTUVWXYZ".ToCharArray();
+
+            return await categories
```

Comment:
I was surprised to see this because it's a db call for every letter of the alphabet

I would have approached it with something like this, but I realise now that you want empty view models for every letter of the alphabet. Also there might be some differences in performance. Mine will querying the entire table in one go which might not be such a great idea 😄 

            return _contentDbContext.GlossaryEntries
                .ToList()
                .GroupBy(e => e.Title[..1].ToUpper(), e => e)
                .Select(e => new GlossaryCategoryViewModel
                {
                    Heading = e.Key,
                    Entries = e.Select(inner => new GlossaryEntryViewModel
                    {
                        Title = inner.Title,
                        Slug = inner.Slug,
                        Body = inner.Body
                    }).OrderBy(inner => inner.Title).ToList()
                })
                .OrderBy(model => model.Heading).ToList();
```

Reply:
I've updated it so it only makes one database call now, but still keeps a element for every letter of the alphabet.